website/markdown/docs/links.mdx

```diff
@@ -73,3 +73,9 @@ import {
 - [Manifesto](/docs/contribution/manifesto/)
 - [Performance](/docs/contribution/performance/)
 - [Generation Pipeline](/docs/contribution/generation-pipeline/)
+- [Website](/docs/contribution/website/)
+
+#### Stats
+
+- [About](/docs/stats/about/)
+- [CommandEvent](/docs/stats/command-event/)
```

Comment:
I'd move this one to the section with the links for contributors. Moreover, I'd rename it because "CommandEvent" is not explicit about what the content of the page it. What about something like "Analytics"?